docs/detections/api/exceptions/api-create-shared-exception-list.asciidoc

```diff
@@ -0,0 +1,86 @@
+[[exceptions-api-create-shared-exception-list]]
+=== Create shared exception list
+
+Creates a shared exception list.
+
+An exception list groups <<exceptions-api-create-exception-item, exception items>>
+and can be associated with rules. When an exception item's query evaluates to
+`true`, rules do *not* issue alerts even when the rule's other criteria are met.
+
+A shared exception list can be utilized across multiple detection rules. 
+For more information, see <<rules-api-create>> and <<rules-api-update>>.
+
+IMPORTANT: All exception items added to the same list are evaluated using
+`OR` logic. That is, if any of the items in a list evaluate to `true`, the
+exception prevents the rule from generating an alert. Likewise, `OR` logic is
+used for evaluating exceptions when more than one exception list is
+assigned to a rule. To use the `AND` operator, you can define multiple clauses
+(`entries`) in a single exception item.
+
+==== Request URL
+
+`POST <kibana host>:<port>/api/exceptions/shared`
+
+For example, using cURL:
+
+[source,console]
+--------------------------------------------------
+curl -X POST "<KibanaURL>/api/exceptions/shared"
+-u <username>:<password> -H 'kbn-xsrf: true' -H "Content-Type: application/json"
+--data "@<link to file>" <1>
+--------------------------------------------------
+<1> The relative link to the `.json` file containing the exception list.
+
+===== Example request
+
+Imports the exception list in the `exception_lists.ndjson` file and overwrites
+existing list with the same `list_id` values:
+
+[source,console]
```

Comment:
Is this example suppose to be identical to the previous one in the Request URL section? If it is, ignore me!

Reply:
hmm yeah it might be redundant having the two. It's possible we could combine them..

Reply:
keeping as is as per convo with Devin in slack